src/newViews/AnalyticalFramework/Canvas/index.tsx

```diff
@@ -179,9 +179,9 @@ function Canvas<T>(props: Props<T>) {
         isSecondary,
         widget: data,
         onWidgetValueChange: handleWidgetValueChange,
-        showWidgetEdit: props.editMode,
+        showWidgetEdit: !props.editMode,
```

Comment:
I maybe wrong, but doesn't this mean, widget should be editable when edit mode is false? Or is it that when editMode is false it means the mode is 'edit mode'? If so, isn't it more practical to change the default value?

Reply:
Yes, editMode is for when a widget is currently being edited.
If any of the widgets are being edited, the edit buttons should be hidden.